infrastructure/tests/test_api_views.py

```diff
@@ -1,59 +1,62 @@
-                "long_name": "City of Cape Town, Western Cape",
-                "square_kms": 2446.42989002681,
-                "parent_level": "province",
-                "parent_code": "WC",
-                "province_name": "Western Cape",
-                "province_code": "WC",
-                "category": "A",
-                "miif_category": "A",
-                "population": 3740026
-            },
-            "function": "Administrative and Corporate Support",
-            "project_description": "Acquisition & Registr & servitude FY20",
-            "project_number": "PC001002004002_00473",
-            "project_type": "New",
-            "mtsf_service_outcome": "An efficient, competitive and responsive economic infrastructure network",
-            "iudf": "Growth",
-            "own_strategic_objectives": "Infrastructure Investment Programme",
-            "asset_class": "Water Supply Infrastructure",
-            "asset_subclass": "Boreholes",
-            "ward_location": "Corporate Infrastructure Projects",
-            "longitude": 0,
-            "latitude": 0
-        }
-        """)
-        self.maxDiff = None
-        response = self.client.get("/api/infrastructure/projects/1/?full=True")
-        self.assertEqual(response.status_code, 200)
+    def setUp(self):
+        fixtures = ["test_infrastructure.json"]
+        TestProject.geography = Geography.objects.create(
+            geo_level="municipality",
+            geo_code="BUF",
+            province_name="Eastern Cape",
+            province_code="EC",
+            category="A",
+        )
 
-        js = response.json()
-        self.assertEquals(test_data, js)
 
-    def test_geography_projects(self):
-        response = self.client.get("/api/infrastructure/projects/?geo=CPT")
+    def test_infrastructure_project_search(self):
+        response = self.client.get("/api/v1/infrastructure/projects/?geo=BUF")
         self.assertEqual(response.status_code, 200)
-        js = response.json()
-        self.assertEquals(len(js["results"]), 2)
-        self.assertEquals(len(js["results"][0]["geography"]["bbox"]), 0)
+        project = Project.objects.create(geography=TestProject.geography)
+        created = utils.create_expenditure(project, "Budget Year 2019/2020", "")
+        self.assertFalse(created)
+        self.assertEquals(Project.objects.all().count(), 1)
 
-        response = self.client.get("/api/infrastructure/projects/?geo=WC011")
-        self.assertEqual(response.status_code, 200)
-        js = response.json()
-        self.assertEquals(len(js["results"]), 1)
+        # response = self.client.get(
+        #     "/api/v1/infrastructure/search/?province=Eastern+Cape&municipality=Buffalo+City&q=&budget_phase=Budget+year&financial_year=2019%2F2020&ordering=-total_forecast_budget")
+        # self.assertEqual(response.status_code, 200)
+        # js = response.json()
+        # self.assertEquals(len(js["results"]), 3)
```

Comment:
this isn't testing search _yet_, right? It's just creating an object and then fetching it, and asserting that the expenditure item was _not_ created?